packages/ui-components/src/components/SelectList/ListItem.tsx

```diff
@@ -66,7 +66,7 @@ const ButtonContainer = styled.div<{
   width: ${({ $fullWidth }) => ($fullWidth ? '100%' : 'auto')};
 `;
 
-const IconWrapper = styled.div`
+const IconWrapper = styled.div<{ $hasIcon?: boolean }>`
```

Comment:
Could potentially get away with accessing `props.children` directly, given how `$hasIcon` is derived. But I can see the readability benefit of this

Happy either way, just thought I’d throw the idea out